discover/portal/articles/210-setting-up/00-intro.markdown

```diff
@@ -0,0 +1,90 @@
+# Setting Up [](id=setting-up)
+
+If @product@ is anything, it's configurable. As the core is shrinking
+due to its increased modularity, it's important that all the applications in
+Liferay are also configurable. 
+
+Breaking it down, three types of applications must be configurable:
+
+1.  The portal itself
+2.  Liferay's native applications 
+3.  Custom applications
+
+To this end, Liferay's engineers have made the platform and its own applications
+configurable, and created a mechanism for developers to make their
+[applications configurable](/develop/tutorials/-/knowledge_base/7-1/configurable-applications).
+
+## Where Configuration Happens [](id=where-configuration-happens)
+
+Liferay's configuration takes place in the following places: 
+
+Properties files: properties files  that set default behavior may be included in
+the platform or the modules. Keep in mind that these settings can always be
+overwritten by a system administrator in the UI. To find what properties are
+configurable this way, visit 
+[docs.liferay.com](https://docs.liferay.com/portal/7.1-latest/propertiesdoc).
+
+Database: configuration through Liferay's UI is stored in a database.  The
```

Comment:
From the perspective of the user I think it is more relevant how they change the configuration than where it's stored.
Considering this, I tend to think that this point should focus on the fact that it's configured through the UI  and mention that the configuration is stored in the database as a comment inside.